ci_scripts/travis/test.sh

```diff
@@ -7,8 +7,7 @@ mkdir -p $TEST_DIR
 cd $TEST_DIR
 
 if [[ "$COVERAGE" == "true" ]]; then
-    py.test -v -x --doctest-modules --ignore=setup.py --pyargs $MODULE \
-        --cov=$MODULE
+    py.test -v -x --doctest-modules --ignore=setup.py --cov=$MODULE
 else
-    py.test -v -x --doctest-modules --ignore=setup.py --pyargs $MODULE
+    py.test -v -x --doctest-modules --ignore=setup.py
```

Comment:
I was a bit confused. Actually pyargs is mandatory to get the path right easily and ignore is superfluous